static/includes/dragonfly-limitations.md

```diff
@@ -1,12 +1,10 @@
-## Migration limitations at General Availability (GA)
+## Migration limitations for General Availability (GA)
 
 As Aiven for Dragonfly moves to General Availability (GA), the focus is on delivering
-the most valuable features to you, guided by your feedback. During the initial phase of
-GA, automatic migration of Users, Access Control Lists (ACLs), and service configurations
-from Redis to Dragonflyare not included.
+valuable features on user feedback. During the initial GA phase, automatic migration of
+users, access control lists (ACLs), and service configurations from Redis to Dragonfly
+is not supported.
 
-What this means:
-
-- **Service configurations:** If you’ve customized your Redis service with
-  specific settings, you must manually apply these settings to Dragonfly. Automatic
-  transfer of these custom configurations is not available during the initial phase of GA.
+If you customized your Aiven for Caching or Aiven for Valkey service with specific
+settings, manually apply these settings to Aiven for Dragonfly. Automatic transfer of
```

Comment:
`during the initial GA phase.` I would drop this part as it's very unlikely this is going to happen. The service configs of valkey and dragonfly are quite different